tomcat/base_test.go

```diff
@@ -34,8 +34,7 @@ import (
 func testBase(t *testing.T, context spec.G, it spec.S) {
 	var (
 		Expect = NewWithT(t).Expect
-
-		ctx libcnb.BuildContext
+		ctx    libcnb.BuildContext
```

Comment:
```suggestion

		ctx    libcnb.BuildContext
```

Reply:
These three suggestions are to just remove format-only code changes. Not strictly necessary, but it keeps the PR slimmer and easier to follow.